src/pykeen/nn/representation.py

```diff
@@ -985,3 +985,31 @@ def _plain_forward(
             labels=[self.labels[i] for i in uniq.tolist()],
         )
         return x[inverse]
+
+
+class WikidataTextRepresentation(LabelBasedTransformerRepresentation):
+    """
+    Textual representations for datasets grounded in Wikidata.
+
+    The textual description are obtained from Wikidata using :class:`WikidataCache`, and encoded with
+    :class:`LabelBasedTransformerRepresentation`.
```

Comment:
can you give a working minimal example? like with codex?

Reply:
https://github.com/pykeen/pykeen/pull/966/commits/c5d10bf532d292328b6cbccad30a1e6b83501391